DESCRIPTION

```diff
@@ -1,13 +1,13 @@
 Package: memoise
-Title: Memoise functions
-Version: 0.2.99
+Title: Memoisation of Functions
+Version: 0.2.1.9000
```

Comment:
I think we need to bump to 0.2.99.9000 so it's a strict increase (and then the release can be 0.3.0 to clean things up)